### ISSUE TITLE
Rethinking the Representational Continuity: Towards Unsupervised Continual Learning

### ISSUE THREAD

**Issue by chaerinkong** [reyllama/MIPAL-CL#4]
## TL; DR

- Unsupervised Continual Learning with SOTA SSL frameworks is a promising direction.
- Features learned from SSL (i.e., Barlow Twins and SimSiam) are more robust to catastrophic forgetting and domain shifts.

## Motivation

- SSL and CL have received much attention, while UCL (SSL+CL) has not. Hence, *bridge the gap between*.

### SimSiam

![image](https://user-images.githubusercontent.com/46308407/158132226-fc12c1d9-1fda-4093-9064-d568a115d725.png)

### Barlow Twins

![image](https://user-images.githubusercontent.com/46308407/158132254-587e13d1-86c1-425e-bd9c-fd7fa36b3e85.png)

### Simple baseline: LUMP

![image](https://user-images.githubusercontent.com/46308407/158132325-6797c8e4-eab5-4bbc-baed-364da509251b.png)

- Achieve competitive performances by simply applying mixup in UCL *(first mover advantage)*

## Evaluation

![image](https://user-images.githubusercontent.com/46308407/158132444-4679fc1f-646b-4e3c-9dc9-890c4d78fc3c.png)
![image](https://user-images.githubusercontent.com/46308407/158132463-860398d6-c327-4e5d-89f7-7df11d92bf4b.png)
![image](https://user-images.githubusercontent.com/46308407/158132542-43264d21-f35f-47e8-bf14-4193ac92bc9f.png)
